lib/vmdb/config/validator.rb

```diff
@@ -1,17 +1,10 @@
 module VMDB
-  class Config
+  module Config
```

Comment:
Same concern as the `Activator`. We should either keep this a class and change the namespacing, or at least change https://github.com/ManageIQ/manageiq/blob/master/lib/vmdb/settings.rb#L55